src/abbfreeathome/devices/rain_sensor.py

```diff
@@ -0,0 +1,83 @@
+"""Free@Home RainSensor Class."""
+
+# I added also the other two output-datapoint, but I let them deacticated.
+# I just don't know when they get changed
+
+from typing import Any
+
+from ..api import FreeAtHomeApi
+from ..bin.pairing import Pairing
+from .base import Base
+
+
+class RainSensor(Base):
+    """Free@Home RainSensor Class."""
+
+    _state_refresh_output_pairings: list[Pairing] = [
+        Pairing.AL_RAIN_ALARM,
+        # Pairing.AL_RAIN_SENSOR_ACTIVATION_PERCENTAGE,
+        # Pairing.AL_RAIN_SENSOR_FREQUENCY,
```

Comment:
Did you mean to leave these commented out? Instead of leaving commented out code maybe it's better to just leave a comment about additional sensors that could be implemented but are not. Keeps the code a bit cleaner.

Reply:
You are right.
I removed the commented code completely. All the other classes also don't have unused pairings included.